script/lock_user

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+if ARGV.empty?
+  puts "Locks a user account so it can't be used."
+  puts "USAGE: script/malicious_user USERNAME"
+  exit
+end
+
+handle = ARGV.first
+puts "Locking user #{handle}..."
+
+ENV["RAILS_ENV"] ||= "production"
+require_relative "../config/environment"
+
+user = User.where(handle: handle).first
```

Comment:
:thinking: Wouldn't be better to wrap the closing logic into method in `User` class, make it tested properly and just call it from this script. Alternatively it can be called from rails console in production if available. => making it first class citizen

I can try to help moving this logic to `User` class if welcomed. Just ping me. Same approach can be done for all scripts in this pull request.

Reply:
I wrote these as scripts because they are specifically for being run from outside a console, with just one argument (like username). The goal is to make it possible to do the thing by running just one command. If you want to send a PR to refactor I won't object, but it doesn't help my needs to have this code in the `User` class. :)